services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy

```diff
@@ -77,22 +75,4 @@ class BaseSpecification extends SpringSpecification implements SetupOnce {
             }.empty
         }
     }
-
-    @HealthCheck
-    def "Create casual flow to ensure Kilda's up state"() {
```

Comment:
I don't see these verifications to be replaced somehow. It used to create a flow and wait for `WAIT_OFFSET * 3` (longer than always) seconds to ensure that Kilda is completely Up after being deployed (**Jenkins case**).
On the other hand, this test was added when we tried to increase stability and I can't claim that it actually increased it. +it may no longer be required since we increased the sleep after kilda deploy from 1 to 3 minutes (but this is still under test and investigation)

Reply:
I think it is just not needed now. In my opinion, if Kilda answers us successful responses when getting all links, switches, flows and link props (all of that are made above), it should be definitely UP. If it is not, so the corresponding bug should be filed I think.